pyvista/core/pointset.py

```diff
@@ -10,7 +10,7 @@
 
 import pyvista
 from pyvista import _vtk
-from pyvista.utilities import abstract_class
+from pyvista.utilities import abstract_class, assert_empty_kwargs
```

Comment:
This addition doesn't seem to be used anywhere.

Reply:
It's not.  I'm removing it.